x/src/commonMain/kotlin/work/socialhub/planetlink/x/action/XAction.kt

```
@@ -0,0 +1,449 @@
+        if (!value.all(Char::isDigit)) {
+            return value
+        }
+        throw NotSupportedException(
+            "kxweb requires an X screen name for this timeline."
+        )
+    }
+
+    private fun parseXUrl(value: String): Url {
+        val url = try {
+            Url(value)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)